skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/routes/tjenestebussintegrasjon/dto/BestillBrevDto.kt

```diff
@@ -0,0 +1,34 @@
+package no.nav.pensjon.brev.skribenten.routes.tjenestebussintegrasjon.dto
+
+import javax.xml.datatype.XMLGregorianCalendar
+
+sealed class BestillBrevResponseDto {
```

Comment:
Jeg synest vi bør gi context til alle request/response objekter slik at de navngis / ligger under en struktur som gjør det tydelig hvilket brev-system det gjelder. Vi kommer til å ha BestillBrevResponse og request objekter for brevbaker, doksys og extream etterhvert.